src/rgw/setup/rgw_service.py

```diff
@@ -31,14 +32,21 @@ def start(conf: MappedConf, config_file, log_file, motr_trace_dir, index: str =
         try:
             os.environ['M0_TRACE_DIR'] = motr_trace_dir
             cmd = "/usr/bin/radosgw"
-            args = f"-f --name client.rgw-{index} -c {config_file} --no-mon-config &> {log_file}"
-            args = shlex.split(args)
+            args = f" -f --name client.rgw-{index} -c {config_file} --no-mon-config &> {log_file}"
             sys.stdout.flush()
             sys.stderr.flush()
-            os.execl(cmd, cmd, *args)  # nosec
+            process = subprocess.Popen(cmd + args, stdin=subprocess.PIPE,
+                                   stdout=subprocess.PIPE, shell=True,
```

Comment:
execl() was consciously kept so current process replaces this process. Why subprocess needed now? subprocess with shell=true is security issue, so we cant add it unless we can prove that its not in this case.
Plus we have SimpleProcess() wrapper class we use everywhere, why subprocess() directly being used here ?